test/packages/false_positives/rally_benchmark_malformed_template/_dev/benchmark/rally/logs-benchmark/template.ndjson

```diff
@@ -1,7 +1,7 @@
 {{- $timestamp := generate "timestamp" -}}
 {
 "@timestamp": "{{ $timestamp.Format "2006-01-02T15:04:05.999999Z07:00" }}",
-"data_stream.type": "logs",
+data_stream.type": "logs",
```

Comment:
As it was this file, `elastic-package test static` did not detect any failure.

With this change, I tried to force a failure here (as mentioned in the README.md of the package).